### PR TITLE
feat(telecom): add softphone nutshell with its module

### PR DESCRIPTION
**Pull request by antonymarion**
ref: MANAGER-13658

| Question         | Answer
| ---------------- | ---
| Branch?          | `master`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Feat#MANAGER-13658
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
~~- [ ] Breaking change is mentioned in relevant commits~~

## Description

This is the nutshell for the new softphone tab inside of the telecom universe

### CODE REVIEW

File: packages/manager/apps/telecom/src/app/telecom/telephony/line/translations/Messages_fr_FR.json [hunk 1]
```diff
@@ -5,6 +5,12 @@
   "telephony_line_tab_consumption": "Consommation",
   "telephony_line_tab_call_management": "Gestion des appels",
   "telephony_line_tab_music_management": "Gestion des musiques",
+  "telephony_line_tab_softphone": "Softphone",
+  "telephony_line_sofpthone_download_links_title": "Liens de téléchargements",
+  "telephony_line_sofpthone_configure_title": "Configurer la ligne",
+  "telephony_line_sofpthone_configure_description": "Vous avez la possibilité de configurer jusqu'à trois appareils au total. Vous pouvez configurer un seul appareil à la fois.",
+  "telephony_line_sofpthone_theme_title": "Appliquer un thème",
+  "telephony_line_sofpthone_download_links_description_label": "Installer l'application Softphone sur vos appareils",
```
Comment (sachinrameshn) [inline packages/manager/apps/telecom/src/app/telecom/telephony/line/translations/Messages_fr_FR.json:13]:
```suggestion
  "telephony_line_softphone_download_links_title": "Liens de téléchargements",
  "telephony_line_softphone_configure_title": "Configurer la ligne",
  "telephony_line_softphone_configure_description": "Vous avez la possibilité de configurer jusqu'à trois appareils au total. Vous pouvez configurer un seul appareil à la fois.",
  "telephony_line_sofpthone_theme_title": "Appliquer un thème",
  "telephony_line_softphone_download_links_description_label": "Installer l'application Softphone sur vos appareils",
```

**Review by sachinrameshn — CHANGES_REQUESTED** [review]

File: packages/manager/apps/telecom/src/app/telecom/telephony/line/softphone/softphone.html [hunk 2]
```diff
@@ -0,0 +1,23 @@
+<oui-tile
+    data-heading="{{ 'telephony_line_sofpthone_download_links_title' | translate }}"
+>
+    <oui-tile-definition
+        data-term="{{ 'telephony_line_sofpthone_download_links_description_label' | translate }}"
```
Comment (sachinrameshn) [inline packages/manager/apps/telecom/src/app/telecom/telephony/line/softphone/softphone.html:5]:
```suggestion
    data-heading="{{ 'telephony_line_softphone_download_links_title' | translate }}"
>
    <oui-tile-definition
        data-term="{{ 'telephony_line_softphone_download_links_description_label' | translate }}"
```

**Review by sachinrameshn — CHANGES_REQUESTED** [review]

**Review by dectotam — APPROVED** [review]

File: packages/manager/apps/telecom/src/app/telecom/telephony/line/softphone/softphone.html [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+        ><span
+            data-translate="telephony_line_softphone_configure_description"
+        ></span
+    ></oui-tile-description>
+</oui-tile>
+<oui-tile
+    class="mt-3"
+    data-heading="{{ 'telephony_line_softphone_theme_title' | translate }}"
+>
+</oui-tile>
```
Comment (darsene) [inline packages/manager/apps/telecom/src/app/telecom/telephony/line/softphone/softphone.html:23]: no content for this tile ? What's the purpose then ?

Comment (antonymarion) [reply packages/manager/apps/telecom/src/app/telecom/telephony/line/softphone/softphone.html:23]: It's a nusthell ;)

File: packages/manager/apps/telecom/src/app/telecom/telephony/line/softphone/softphone.html [hunk 4]
```diff
@@ -0,0 +1,23 @@
+<oui-tile
+    data-heading="{{ 'telephony_line_softphone_download_links_title' | translate }}"
+>
+    <oui-tile-definition
+        data-term="{{ 'telephony_line_softphone_download_links_description_label' | translate }}"
+    >
+    </oui-tile-definition>
+</oui-tile>
```
Comment (ghyenne) [inline packages/manager/apps/telecom/src/app/telecom/telephony/line/softphone/softphone.html:8]: For the download part it ask an oui-collapsible not a tile.

**Review by darsene — APPROVED** [review]

File: packages/manager/apps/telecom/src/app/telecom/telephony/line/softphone/softphone.routing.js [hunk 5]
```diff
@@ -0,0 +1,21 @@
+import template from './softphone.html';
+
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state(
+    'telecom.telephony.billingAccount.line.dashboard.softphone',
+    {
+      url: '/softphone',
+      views: {
+        'lineInnerView@telecom.telephony.billingAccount.line.dashboard': {
+          template,
```
Comment (ghyenne) [inline packages/manager/apps/telecom/src/app/telecom/telephony/line/softphone/softphone.routing.js:10]: Should be an component not directly a template.

File: packages/manager/apps/telecom/src/app/telecom/telephony/line/softphone/softphone.html (hunk 4 above)
Comment (antonymarion) [reply packages/manager/apps/telecom/src/app/telecom/telephony/line/softphone/softphone.html]: Done, use oui collapsible everywhere with custom one (not clickable)

File: packages/manager/apps/telecom/src/app/telecom/telephony/line/softphone/softphone.html [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+    data-heading="{{ 'telephony_line_softphone_download_links_title' | translate }}"
+>
+    <span
+        data-translate="telephony_line_softphone_download_links_description_label"
+    >
+    </span>
+</oui-collapsible>
+<oui-collapsible
+    class="custom-collapsible-no-click mt-3"
+    expanded="true"
```
Comment (ghyenne) [inline packages/manager/apps/telecom/src/app/telecom/telephony/line/softphone/softphone.html:12]:
```suggestion
    data-expanded="true"
```

File: packages/manager/apps/telecom/src/app/telecom/telephony/line/softphone/softphone.html [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+    data-heading="{{ 'telephony_line_softphone_configure_title' | translate }}"
+>
+    <span
+        data-translate="telephony_line_softphone_configure_description"
+    ></span>
+</oui-collapsible>
+<oui-collapsible
+    class="custom-collapsible-no-click mt-3"
+    expanded="true"
```
Comment (ghyenne) [inline packages/manager/apps/telecom/src/app/telecom/telephony/line/softphone/softphone.html:21]:
```suggestion
    data-expanded="true"
```

**Review by anooparveti — APPROVED** [review]

**Pull request merged**